src/main/java/cam72cam/immersiverailroading/physics/PhysicsAccummulator.java

```diff
@@ -35,55 +38,63 @@ public void accumulate(EntityRollingStock stock, Boolean direction) {
 		}
 		
 		EntityMoveableRollingStock movable = ((EntityMoveableRollingStock)stock);
-		
-		// SHOULD THIS HAVE DIRECTION MULT?
+
 		double stockMassLb = 2.20462 * stock.getWeight();
-		rollingResistanceNewtons += 0.0015 * stockMassLb * 4.44822f;
-		
-		// SHOULD THIS HAVE DIRECTION MULT?
-		double grade = -Math.tan(Math.toRadians(pos.rotationPitch % 90)) * Config.ConfigBalance.slopeMultiplier;
-		// lbs * 1%gradeResistance * grade multiplier
-		gradeForceNewtons += (stockMassLb / 100) * (grade * 100)  * 4.44822f;
+		rollingResistanceNewtons += 0.0015 * stockMassLb * poundToNewton;
+
+		// see http://hyperphysics.phy-astr.gsu.edu/hbase/mincl.html for slope physics
+		// negative because a positive rotationPitch means stock is pointing uphill
+		double gradeForceFactor = -Math.sin(Math.toRadians(pos.rotationPitch)) * Config.ConfigBalance.slopeMultiplier;
```

Comment:
This seems to be doing weird things.  Can you try this out locally on a few different slopes?  It seems to be accelerating in weird ways

Reply:
Actually, my settings are tweaked.  let me reset them and try again